tests/integration/replica-redirect.tcl

```diff
@@ -19,6 +19,11 @@ start_server {tags {needs:repl external:skip}} {
             r replicaof $primary_host $primary_port
             wait_replica_online $primary
 
+            # In case of bio thread RDB download, there can be up to 1000ms 
+            # (1 replication cron loop) delay until the rdb starts loading
+            after 1000 
```

Comment:
Why is this necessary? the next `wait_done_loading` isn't enough?

Reply:
The way `wait_done_loading` works is by polling PING to detect when Valkey becomes responsive, which only happens after the RDB loading process has started. However, there can be up to a 1000ms delay before the loading even begins - because the replication cron that triggers the RDB load runs asynchronously, and we can't predict exactly when its next iteration will happen. Without the after 1000, `wait_done_loading` might start too early and incorrectly conclude that the server is already ready when it hasn't started loading yet.